README.md

```diff
@@ -51,6 +51,12 @@ either through custom URI scheme redirects, or App Links.
 AS's that assume all clients are web-based or require clients to maintain
 confidentiality of the client secrets may not work well.
 
+From Android API 30 (R) and above, set [QUERY_ALL_PACKAGES](https://developer.android.com/reference/kotlin/android/Manifest.permission#query_all_packages) in the manifest,
```

Comment:
This permission will most likely not be allowed for common apps (not launcher, not security related, etc) in the Play store. An alternative might be:

```
    <queries>
        <intent>
            <action android:name="android.intent.action.VIEW" />
            <data android:mimeType="*/*" />
        </intent>
    </queries>
```

However, it remains to be seen if Google will allow this for apps published in the Play store.

Reply:
Updated in the newest commit, limited to what we actually need.
See https://github.com/openid/AppAuth-Android/issues/554